builder.toml

```diff
@@ -4,7 +4,7 @@ build-image = "heroku/pack:18-build"
 run-image = "heroku/pack:18"
 
 [lifecycle]
-version = "0.5.0"
+version = "0.6.1"
```

Comment:
Will bumping this break any buildpacks that exist outside of our builders that we know of?

Reply:
not that i know of. but it could. probably more reason to version the image beyond `:18`

Reply:
I like the idea of having versions beyond `:18`, it would make us simply fallback to previous "version" of the package, which we are not able to do now :(